src/certificates/layout/MainLayout.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Container, Button, Layout, Dropdown, DropdownButton,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import SubHeader from '../../generic/sub-header/SubHeader';
+import CertificatesSidebar from '../certificates-sidebar';
+import messages from '../messages';
+
+const MainLayout = ({ courseId, showHeaderButtons, children }) => {
+  const intl = useIntl();
+  return (
+    <Container size="xl" className="certificates px-4">
+      <div className="mt-5" />
+      <SubHeader
+        subtitle={intl.formatMessage(messages.headingSubtitle)}
+        title={intl.formatMessage(messages.headingTitle)}
+        headerActions={showHeaderButtons && ( // TODO create btn handlers in the task (https://youtrack.raccoongang.com/issue/AXIMST-166)
```

Comment:
[important] Judging by the design, the buttons in the `SubHeader` component should be located along the bottom border of the text to the right. Please make sure of this

**Design**
<img width="1069" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/e5d7e445-b793-461f-89a1-c4bccfd2e52b">

**Current implementation**
![image](https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/867100d2-d7bb-437a-8afa-6da3d2a5efaf)